sdk/search/search-documents/CHANGELOG.md

```diff
@@ -9,7 +9,7 @@
 
 ### Changes since 11.1.0-beta.2
 
-- The `@azure/search-documents` has been moved to V6 SDK Generator. The following types `BlobIndexerDataToExtract`, `BlobIndexerImageAction`, `BlobIndexerParsingMode`, `BlobIndexerPDFTextRotationAlgorithm` has been changed to `string`. Please refer [#12829](https://github.com/Azure/azure-sdk-for-js/pull/12829) for further details.
+- The types `BlobIndexerDataToExtract`, `BlobIndexerImageAction`, `BlobIndexerParsingMode`, `BlobIndexerPDFTextRotationAlgorithm` have changed from union of string literals to string. This is to support the service definition for these types to be extensible enums. The documentation on methods that use these have been updated with known values that can be used for these types. Please refer [#12829](https://github.com/Azure/azure-sdk-for-js/pull/12829) for further details.
```

Comment:
minor nit because devs might not know what an 'extensible enum' is (since enums are imaginary in JS):

```suggestion
- The types `BlobIndexerDataToExtract`, `BlobIndexerImageAction`, `BlobIndexerParsingMode`, `BlobIndexerPDFTextRotationAlgorithm` have changed from union of string literals to string. This is to allow the service to support additional values after the release of this SDK. The documentation on methods that use these types have been updated with currently known values. Please refer [#12829](https://github.com/Azure/azure-sdk-for-js/pull/12829) for further details.
```